benchmarks/2022/benchmarks-schema.yaml

```diff
@@ -16,6 +16,12 @@ definitions:
       isToppedOut:
         description: isToppedOut is a boolean value that describes whether a benchmark's latter deciles repeat a value of 100 or, in the case of inverse measures, a value of 0.
         type: boolean
+      isToppedOutByProgram:
+        description: isToppedOutByProgram is a boolean value that describes whether a benchmark's latter deciles repeat a value of 100 or, in the case of inverse measures, a value of 0.
```

Comment:
I need a better description here; reaching out to Michelle/Kyle

Reply:
Topped-out measure is one whose median performance score is 95% or higher and whose performance is “so high and unvarying that meaningful distinctions and improvement in performance can no longer be made.” These measurements scores are capped at 7 points.